include/hip/hcc_detail/texture_functions.h

```diff
@@ -23,14 +23,24 @@ THE SOFTWARE.
 #ifndef HIP_INCLUDE_HIP_HCC_DETAIL_TEXTURE_FUNCTIONS_H
 #define HIP_INCLUDE_HIP_HCC_DETAIL_TEXTURE_FUNCTIONS_H
 
-#include <hc.hpp>
-#include <hc_short_vector.hpp>
+#include <hip/hcc_detail/hip_vector_types.h>
 #include <hip/hcc_detail/hip_texture_types.h>
 
+#define TYPEDEF_VECTOR_VALUE_TYPE(SCALAR_TYPE) \
```

Comment:
Please use #pragma push_macro/pop_macro to protect/hide TYPEDEF_VECTOR_VALUE_TYPE

Reply:
Thanks Sam, I just added a pragma to protect it